### PR TITLE
Example using DT

### PR DESCRIPTION
**Pull request by JoergBruenner**
### Checklist

- [x] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/github-templates/CONTRIBUTING.md#pull-requests)
- [ ] This PR fixes a defect, and I have provided tests to verify that the fix is effective
- [ ] This PR implements an enhancement, and I have provided tests to verify that it works as intended
- [ ] This PR introduces changes to the database model, and I have added corresponding [update logic](https://github.com/DependencyTrack/dependency-track/tree/master/src/main/java/org/dependencytrack/upgrade)
- [ ] This PR introduces new or alters existing behavior, and I have updated the [documentation](https://github.com/DependencyTrack/dependency-track/tree/master/docs/_docs) accordingly

### CODE REVIEW

**Comment by nscuro** [comment]
I think this is an amazing idea, thanks @JoergBruenner! 🙌

We'll need you to sign-off your commit though so the DCO check passes. If you click on the *Details* link of the check, it'll show you what you can do to resolve this.

**Comment by JoergBruenner** [comment]
Hi Niklas,you may guess, I'm not a frequent Git user. So I didn't got the message. Does "Sign off" means digitally sign it? I'm using Github Desktop. All the hints I've found seems to be made for other tools???Thank you for your helpJörg
-------- Ursprüngliche Nachricht --------Von: Niklas ***@***.***> Datum: 01.12.22  15:46  (GMT+01:00) An: DependencyTrack/dependency-track ***@***.***> Cc: JoergBruenner ***@***.***>, Mention ***@***.***> Betreff: Re: [DependencyTrack/dependency-track] Example using DT (PR #2211) 

I think this is an amazing idea, thanks @JoergBruenner! 🙌

We'll need you to sign-off your commit though so the DCO check passes. If you click on the Details link of the check, it'll show you what you can do to resolve this.

File: docs/_docs/usage/usecases.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+title: Use Cases
+category: Usage
+chapter: 2
+order: 6
+---
+
+Here we provide some practical insights. Sometimes it's hard to begin something new. We don't think usage of Dependency Track shall prove this.
+
+> We provide Windows Powershell Scripts. This are working examples but they aren't ready for automated productive use.
+> Add rights management, error management, monitoring etc. to fit your environment.
```
Comment (valentijnscholten) [inline docs/_docs/usage/usecases.md:11]:
```suggestion
Here we provide some practical examples on how to use Dependency Track and its API. These are working examples but please add rights management, error management, monitoring etc. to fit your environment.
```

**Comment by nscuro** [comment]
> @JoergBruenner Does "Sign off" means digitally sign it? I'm using Github Desktop. All the hints I've found seems to be made for other tools???

Sign-off in Git is referring to the [Developer Certificate of Origin](https://developercertificate.org/) (DCO). Most Open Source projects, including Dependency-Track, require all contributors to include acknowledgement of the DCO in all of their commits. This is documented in our [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#pull-requests).

One the command line, you can use the `-s` or `--signoff` flag *for new commits*, e.g. 
```shell
git commit --signoff -m "My commit"`
```

For existing commits, like in your current case, sign-off requires some extra steps. As mentioned before, our [DCO build check](https://github.com/DependencyTrack/dependency-track/pull/2211/checks?check_run_id=9876349402) shows you what to do:

> To add your Signed-off-by line to every commit in this branch:
> 1. Ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally).
> 2. In your local branch, run: git rebase HEAD~3 --signoff
> 3. Force push your changes to overwrite the branch: git push --force-with-lease origin master

**Review by nscuro — APPROVED** [review]
Thanks @JoergBruenner!

**Pull request merged**